dialect/sql/schema/postgres.go

```diff
@@ -87,7 +87,7 @@ func (d *Postgres) setRange(ctx context.Context, tx dialect.Tx, t *Table, value
 func (d *Postgres) table(ctx context.Context, tx dialect.Tx, name string) (*Table, error) {
 	rows := &sql.Rows{}
 	query, args := sql.Dialect(dialect.Postgres).
-		Select("column_name", "data_type", "is_nullable", "column_default").
+		Select("column_name", "data_type", "is_nullable", "column_default", "udt_name").
```

Comment:
Do you need this column for future PRs? because you don't use it atm.

Reply:
Yes. it used in any future feature that needs access to the custom type. E.g. composite types or enums